### PR TITLE
ext: Checkout rav1e v0.6.6

### PR DESCRIPTION
**Pull request by wantehchang**

### CODE REVIEW

**Review by vigneshvg — APPROVED** [review]
The appveyor build failed on this PR. But it was fixed here: https://github.com/AOMediaCodec/libavif/pull/1409

File: tests/data/README.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ This is allowed per the specification. libavif should decode such files
 correctly (i.e.) it should report them as files with alpha channel.
 
 Box structure of the items in this file:
+
```
Comment (jzern) [inline tests/data/README.md:209]: This seems unrelated, was it meant for this change?

Comment (wantehchang) [reply tests/data/README.md:209]: Yes, this is an unrelated change and will be merged as a separate commit. I added this commit to this pull request to force the AppVeyor build to run again. This fixes the rendering of this README.md file inside Google.

Please review this commit as an independent change. Thanks.

Comment (vigneshvg) [reply tests/data/README.md:209]: Ack, please merge this PR without squashing the commits.

**Review by vigneshvg — APPROVED** [review]
